tests/Modules/FastestCp.Tests/FastestCpControllerTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿using EvoSC.Common.Remote.EventArgsModels;
+using EvoSC.Modules.Official.FastestCp.Controllers;
+using EvoSC.Modules.Official.FastestCp.Interfaces;
+using GbxRemoteNet.Events;
+using Moq;
+
+namespace EvoSC.Modules.Official.FastestCp.Tests;
+
+public class FastestCpControllerTest
+{
+    private readonly FastestCpController _fastestCpController;
+    private readonly Mock<IFastestCpService> _fastestCpService = new();
+
+    public FastestCpControllerTest()
+    {
+        _fastestCpController = new FastestCpController(_fastestCpService.Object);
+    }
+
+    [Fact]
+    public async void ShouldRegisterTimeAndShowWidget()

```

Comment:
```suggestion
    public async void Should_Register_Time_And_Show_Widget()
```

To keep consistency, we should stick with snake case.